### ISSUE TITLE
simtex gets path to images wrong.

### ISSUE THREAD

**Issue by johnblommers** [iaacornus/simtex#4]
Using the sample markdown file it refers to `./sample_image.jpeg` which is in the same directory. However while simtex processes the sample markdown file we get an error message:

```bash
INFO     Copying ./sample_image.jpeg into Example.tex ...                                                
ERROR    Encountered: [Errno 2] No such file or directory: '/sample_image.jpeg' while moving             
         ./sample_image.jpeg, skipping   
```

Commandline:

```bash
simtex -c -i=example.md -T="Example Simtex" -of="." -f="Example.tex" -a="Johannes Rexx" -d="Wed. Aug 17, 2022"
```

Contents of `~/.config/simtex/simtex.json` is

```json
[
    {
        "FOR": [
                "markdown",
                "md",
                "rst",
                "txt",
                "text"
            ],
        "CODE_BLOCKS": "```",
        "IMAGE": "!\\[([^]]+)\\]\\(([^]]+)\\)",
        "LINKS": "\\[([^]]+)\\]\\(([^]]+)\\)",
        "SECTION": "#",
        "SUBSECTION": "##",
        "SUBSUBSECTION": "###",
        "PARAGRAPH": "####",
        "SUBPARAGRAPH": "#####",
        "INLINE_MATH": "$",
        "PARAGRAPH_MATH": "$$",
        "INLINE_CODE": "`(.*?)`"
    },
    {
        "DOC_CLASS": "article",
        "DEF_FONT": "lmodern",
        "FONT_SIZE": 12,
        "MARGIN": 1,
        "PAPER_SIZE": "a4paper",
        "INDENT_SIZE": 24,
        "SLOPPY": true,
        "CODE_FONT": "DejaVuSansMono",
        "CFONT_SCALE": 0.9,
        "CODE_CONF": "/home/johnb/Applications/SimTeX/code_conf.txt",
        "PACKAGES": [
                "[margin=<MARGIN>, <PAPER_SIZE>]{geometry}",
                "{indentfirst}",
                "{amsmath}",
                "{mathtools}",
                "{sectsty}",
                "{footmisc}",
                "{gensymb}",
                "{xcolor}",
                "{listings}",
                "{caption}",
                "[colorlinks, allcolors=<LINK_COLORS>]{hyperref}"
            ],
        "FOOTNOTE": "footnote",
        "SECTION_SIZES": {
                "main": 12,
                "sub": 12,
                "subsub": 12
            },
        "LINKS": true,
        "LINK_COLOR": "blue",
        "AUTHOR": "Johannes Rexx",
        "DATE": "<NOW>",
        "MAKE_TITLE": true,
        "FILE_NAME": "output.tex",
        "OUTPUT_FOLDER": "./out"
    }
]
```

The good news is that the `Example.tex` output file is correctly produced in the same directory as the `example.md` and `sample_image.jpeg`.

**Comment by iaacornus** [comment]
thank you for reporting, it was due to the error in the way that the original path of image is determined which was addressed in the mentioned commits. I'll close this later after final testing for the stable, since those commits aren't thoroughly tested and there isn't test case for this yet.

**Comment by johnblommers** [comment]
It was also necessary to use `.` as the folder to write the output file in because if we set the output folder to `out` then TeX can't find the image, which is located one directory up.

**Comment by iaacornus** [comment]
i'm closing this as it was fixed since 2.0 and was tested in 2.1
